File: app/src/main/java/bobrchess/of/by/belaruschess/presenter/callback/CallBackCountry.java
package bobrchess.of.by.belaruschess.presenter.callback;

import java.util.List;

import bobrchess.of.by.belaruschess.dto.CountryDTO;

/**
 * Created by Igor on 11.04.2018.
 */

public interface CallBackCountry extends CallBack {
    void onResponse(CountryDTO countryDTO);

    void onResponse(List<CountryDTO> countryDTO);
}

File: app/src/main/java/bobrchess/of/by/belaruschess/presenter/callback/CallBackCountry.kt
package bobrchess.of.by.belaruschess.presenter.callback

import bobrchess.of.by.belaruschess.dto.CountryDTO

/**
 * Created by Igor on 11.04.2018.
 */

interface CallBackCountry : CallBack {
    fun onResponse(countryDTO: CountryDTO)

    fun onResponse(countryDTO: List<CountryDTO>)
}
